stories/index.stories.js

```diff
@@ -243,4 +245,24 @@ storiesOf('TreeMenu', module)
         )}
       </TreeMenu>
     );
-  });
+  })
+  .add('Opened/Closed Icon', () => (
+    <TreeMenu 
+      data={dataInArray} 
+      onClickItem={action(`on click node`)}       
+      >
+        {({ items }) => (          
+          <ul className="tree-item-group">     
+            {items.map(props => ( <ItemComponent {...props} openedIcon={openedIcon} closedIcon={closedIcon} /> ))}
+          </ul>
+        )}
+      </TreeMenu>
+  )); 
+
```

Comment:
`openedIcon` and `closedIcon` should be declared before usage, not after. And you should get an error `Warning: Functions are not valid as a React child. This may happen if you return a Component instead of <Component /> from render. Or maybe you meant to call this function rather than return it.` when you run `yarn storybook`. As I said earlier, you only need a ReactNode, not a function, for example:
```
const openedIcon = <img src={closedIconImg} alt="-"></img>;
```